x-pack/plugins/security_solution/public/resolver/store/selectors.ts

```diff
@@ -359,6 +366,16 @@ export const isLoadingMoreNodeEventsInCategory = composeSelectors(
   dataSelectors.isLoadingMoreNodeEventsInCategory
 );
 
+export const eventsInCategoryResultIsStale = composeSelectors(
```

Comment:
@jonathan-buttner - just a reminder. We'll probably need to add a stale check for nodeData now since that is a separate request now